docs/guides/developer/mkdocs.yml

```diff
@@ -60,9 +61,7 @@ pages:
         - Dropdowns: 'modules/admin-ui/style/dropdowns.md'
         - Alerts and Indicators: 'modules/admin-ui/style/alerts-indicators.md'
         - References: 'modules/admin-ui/style/references.md'
-   - Capture Agent:
-     - Capture Agent: 'modules/capture-agent/capture-agent.md'
-     - Test Cases: 'modules/capture-agent/testcases.md'
```

Comment:
You accidentally deleted a few other entries.
Can you please fix that?
Everything else looks fine now.